octoprint_Spoolman/SpoolmanPlugin.py

```diff
@@ -68,7 +69,8 @@ def on_event(self, event, payload):
             event == Events.PRINT_CANCELLED
         ):
             self.handlePrintingStatusChange(event)
-
+        elif event == Events.Z_CHANGE:
```

Comment:
Is it possible that `Z_CHANGE` event gets triggered outside of printing? Eg. when user jogs the toolhead up for some reason, or when filament loading / unloading procedure is being performed? In such cases, such an event should be ignored, as there's no print that needs to be handled yet.